specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -695,9 +706,12 @@ def verify_cell_kzg_proof_batch(commitments_bytes: Sequence[Bytes48],
         assert len(proof_bytes) == BYTES_PER_PROOF
 
     # Get objects from bytes
+    # Create the list of unique commitments we are dealing with
     deduplicated_commitments = list(set(commitments_bytes))
```

Comment:
I think we should remove the "Get objects from bytes" comment. Doesn't add all that much there.

Reply:
Addressed in e685f66bf